api/src/db/migrations/versions/2025_08_23_add_deleted_status_to_sam_gov_processing_status.py

```diff
@@ -0,0 +1,40 @@
+"""add_deleted_status_to_sam_gov_processing_status
+
+Revision ID: f89e8dd701ad
+Revises: e79e8dd701ac
+Create Date: 2025-08-23 10:00:00.000000
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f89e8dd701ad"
+down_revision = "e79e8dd701ac"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # Insert the new DELETED status into the lookup table
+    op.execute(
+        """
+        INSERT INTO api.lk_sam_gov_processing_status (sam_gov_processing_status_id, description, created_at, updated_at)
+        VALUES (4, 'deleted', NOW(), NOW())
+        ON CONFLICT (sam_gov_processing_status_id) DO NOTHING
+        """
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # Remove the DELETED status from the lookup table
+    op.execute(
+        """
+        DELETE FROM api.lk_sam_gov_processing_status
+        WHERE sam_gov_processing_status_id = 4 AND description = 'deleted'
+        """
+    )
```

Comment:
We do not need to use a migration to add new lookup values. The migration process already sync lookup values. By just adding the `LookupStr` to the list of values, running `make db-migrate` will automatically do basically this.

`make db-migrate` calls this function:
* https://github.com/HHS/simpler-grants-gov/blob/main/api/src/db/migrations/run.py#L25
* which in turn calls https://github.com/HHS/simpler-grants-gov/blob/main/api/src/db/models/lookup/sync_lookup_values.py#L12

Which is hooked into the whole setup for how lookup values work.


```suggestion

```